<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QWidget" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hand gesture recognition</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="trainButton">
       <property name="text">
        <string>Train Network</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="perfButton">
       <property name="text">
        <string>Network Performances</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="liveTestButton">
       <property name="text">
        <string>Live Camera Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
